assets/stylesheets/components/_tables.scss

```diff
@@ -0,0 +1,105 @@
+// .table-header in _headings.scss
+
+table {
```

Comment:
Bootstrap uses a `.table` selector for any global declarations. By adding `width: 100%` to the global element, you're impacting all tables; I don't think we can assume that all tables, organization-wide, will always be suited for 100% width. 

Can we please implement `.table` and override any BS defaults so that you get what you want? We'll need to also update the styleguide implementations with this convention.